app/build.gradle.kts

```diff
@@ -87,6 +88,10 @@ dependencies {
     implementation(project(":features:sketch"))
     implementation(project(":core:ui-common"))
 
+    // kover
+    kover(project(":features:sketch"))
+    kover(project(":core:ui-common"))
```

Comment:
What does it to exactly?

`kover` will join all the coverage reports from these dependencies as if they were part of `app`?

----

In any case, we could maybe simplify it with something like:

```kotlin
dependencies {
    fun localModule(name: String) = project(named).also {
        kover(it)
    }

    implementation(localModule(":features:sketch"))
    implementation(localModule(":core:ui-common"))
}
```

----

Do we need to add all modules to `kover` like that?
I foresee an issue in the future, where we might have modules that are _not_ a dependency to `app`.

For example, we wouldn't have coverage report to `audio:processing` in the example below:

```mermaid
flowchart TD
    App --> Core
    App --> audio:recording
    audio:recording --> Core
    audio:recording --> audio:processing
```

Or we would need to add `kover(project(":audio:processing")` in `audio:recording`'s `build.gradle` file?

I'm not sure if it's elegant to keep adding random modules to `:app` even if they are not dependencies, just for coverage purposes 🤔 
Can't we configure `kover` on the top-level? Or on each module independently?

I really don't care much about it now, but something for the future, really.

Reply:
Maybe we can come up with something smarter later on.

Something like:

```kotlin
val projectsWithKover = rootProject.subprojects.filter {
    plugins.any { it is KoverGradlePlugin }
}
projectsWithKover.forEach {
    kover(it.path)
}
```

Reply:
Yes, that's a good point, and didn't think about it. 

I think we might come up with a better setup in the future, I was actually looking at `kover`, and it will change its model in the newer releases [starting 0.8.x](https://github.com/Kotlin/kotlinx-kover/blob/main/kover-gradle-plugin/docs/migrations/migration-to-0.8.0.md). So that sounds maybe like the best moment to address the improved config.

For now, it is how you described, to add all reports to app. 🙃